articles/setup-ssh-ubuntu-vm-aws/index.md

```diff
@@ -0,0 +1,120 @@
+
+### Launch an Amazon EC2 Virtual Machine
+1. Navigate to the [AWS console](https://console.aws.amazon.com/).
+2. On the navbar, navigate to Services > Compute > EC2. This will take you to the EC2 console.
+3. Scroll down to the "Launch Instance" prompt and click on it.
+4. You will then navigate to a page where you get to choose an Amazon Machine Image.
+5. Under QuickStart, you should select the Ubuntu Server 18.04 LTS that is Free tier eligible.
+![Choose an Amazon Machine Image](/engineering-education/setup-ssh-ubuntu-vm-aws/ami.png)
+6. Navigate to the tab where you select the Instance type. For this case, a General Purpose t3.micro instance that is Free tier eligible is sufficient.
+![Select Instance Type](/engineering-education/setup-ssh-ubuntu-vm-aws/instance_type.png)
+7. Click the button labeled "Next: Configure Instance Details".
+8. On the next three subsequent pages, you do not need to make any changes. (Click the buttons labelled "Next: Add storage", "Next: Add Tags" and "Configure Security Group" respectively.)
+9. On the Configure Security Group page, select create a new security group and leave everything else as is. (Ignore the displayed warning.) Click "Review and Launch".
+![Configure Security Group](/engineering-education/setup-ssh-ubuntu-vm-aws/configure_security_group.png)
+10. On the Launch Page, click on "Launch", which will trigger a prompt. Select "create a new key pair". Name the key pair and download it. (Later on, you will use your_pem_file.pem to SSH into the Virtual Machine instance you are about to launch.) When done, you can then click on "Launch Instances".
+![Launch Instance](/engineering-education/setup-ssh-ubuntu-vm-aws/launch_instance.png)
+
+**CONGRATULATIONS!!** You have managed to set up and launch your own Virtual Machine Instance.
+Now you can move on to connecting to it remotely via the [SSH protocol](https://www.ssh.com/ssh/protocol/).
+
+### SSH into the Amazon EC2 Virtual Machine
+The operating system of your local computer determines the options available to SSH from your local computer to your Virtual Machine instance running Linux (Ubuntu 18.04).
+
+Moving forward, the tutorial assumes your local computer operating system is Linux or macOS X. If this isn't the case, see [Amazon's guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstances.html?icmpid=docs_ec2_console).
+
+First, you need to locate your private key file (your_pem_file.pem) that you downloaded earlier, and move it to the `~/.ssh` directory. To do this, from the terminal in the directory where your_pem_file.pem was downloaded, run:
+
+```bash
+mv your_pem_file.pem ~/.ssh/your_pem_file.pem
+```
+
+Change from your current directory to the `~/.ssh` directory where you placed the private key file:
+
+```bash
+cd ~/.ssh
+```
+
+Next, you need to ensure your private key file (your_pem_file.pem) is not publicly viewable for SSH to work. To do this, run:
+
+```bash
+chmod 400 ubuntu.pem
+```
```

Comment:
You don't really need to `cd` into `~/.ssh`.  You can simply `chmod 400 ~/.ssh/your_pem_file.pem`.  All in one step.

Reply:
Noted 😄